core/stream/convert.go

```diff
@@ -373,13 +373,34 @@ func ftou(count int, dst, src buf) error {
 	dstBits, srcBits := dstTy.Bits(), srcTy.Bits()
 	dstMask := (1 << dstBits) - 1
 	ss := binary.BitStream{Data: src.bytes, ReadPos: src.offset}
+	ss2 := binary.BitStream{Data: src.bytes, ReadPos: src.offset}
 	ds := binary.BitStream{Data: dst.bytes, WritePos: dst.offset}
 	switch {
 	case srcIsF16:
 		scale := float32(dstMask)
+		min, max := float32(math.MaxFloat32), float32(0)
```

Comment:
Would there be a case where all the iterated f are negative? if that then the max initialization should be -float32(math.MaxFloat32)?